Assets/SEE/UI/Window/MetricWindow.cs

```diff
@@ -0,0 +1,188 @@
+        public void CreateUIInstance()
+        {
+            //Instantiate MetricWindow
+            MetricWindowObject = PrefabInstantiator.InstantiatePrefab(SettingsPrefab, Window.transform.Find("Content"), false);
+            MetricWindowObject.name = "Scrollable";
+
+            //Parent Content
+            Transform ScrollViewContent = MetricWindowObject.transform.Find("Content/Items").transform;
+
+            //Input Field
+            TMP_InputField inputField = MetricWindowObject.transform.Find("Search/SearchField").gameObject.MustGetComponent<TMP_InputField>();
+
+            inputField.onSelect.AddListener(str => SEEInput.KeyboardShortcutsEnabled = false);
+            inputField.onDeselect.AddListener(str => SEEInput.KeyboardShortcutsEnabled = true);
+
+            //Int Attributes
+            foreach (KeyValuePair<string, int> kvp in graphElement.IntAttributes)
+            {
+                //Create GameObject
+                itemRow = PrefabInstantiator.InstantiatePrefab(itemPrefab, ScrollViewContent, false);
+                //Attribute Name
+                TextMeshProUGUI attributeTextClone = itemRow.transform.Find("AttributeLine").gameObject.MustGetComponent<TextMeshProUGUI>();
+                attributeTextClone.text = kvp.Key;
+                //Value Name
+                TextMeshProUGUI valueTextClone = itemRow.transform.Find("ValueLine").gameObject.MustGetComponent<TextMeshProUGUI>();
+                valueTextClone.text = kvp.Value.ToString();
+            }
+
+            //Float Attributes
+            foreach (KeyValuePair<string, float> kvp in graphElement.FloatAttributes)
+            {
+                //Create GameObject
+                itemRow = PrefabInstantiator.InstantiatePrefab(itemPrefab, ScrollViewContent, false);
+                //Attribute Name
+                TextMeshProUGUI attributeTextClone = itemRow.transform.Find("AttributeLine").gameObject.MustGetComponent<TextMeshProUGUI>();
+                attributeTextClone.text = kvp.Key;
+                //Value Name
+                TextMeshProUGUI valueTextClone = itemRow.transform.Find("ValueLine").gameObject.MustGetComponent<TextMeshProUGUI>();
+                valueTextClone.text = kvp.Value.ToString();
+            }
+
+            //Save GameObjects in Array for SearchField
+            int totalElements = ScrollViewContent.transform.childCount;
+            GameObject[] Element = new GameObject[totalElements];
+
+            for (int i = 0; i < totalElements; i++)
+            {
+                Element[i] = ScrollViewContent.transform.GetChild(i).gameObject;
+            }
+
+            inputField.onValueChanged.AddListener(str => InputSearchField(str, Element));
+        }
+
+        public override void RebuildLayout()
+        {
+            // Nothing needs to be done.
+        }
+
+        protected override void InitializeFromValueObject(WindowValues valueObject)
+        {
+            throw new NotImplementedException();
```

Comment:
Is it okay that his method is not implemented? If so, please add a comment.

Reply:
Every TODO must refer to an issue number.

Reply:
fixed